tools/perform_upgrade.sh

```diff
@@ -22,8 +22,8 @@ set -e
 
 BOARD="$1"
 
-./deploy.py --board=$BOARD --opensk --programmer=none
-python3 -m tools.deploy_partition --board=$BOARD
+./deploy.py --board=${BOARD} --opensk --programmer=none $2
```

Comment:
The double quotes are necessary because you don't want the variable to expand to multiple words.

```
BOARD='foo bar'
"${BOARD}"  # becomes ["foo bar"] with Rust-like syntax
${BOARD}  # becomes ["foo", "bar"] with Rust-like syntax
```